README.md

```diff
@@ -4,10 +4,19 @@ An AWS S3 storage adapter for Ghost 0.10+
 
 ## Installation
 
+```bash
+# Install the module
+npm install ghost-storage-adapter-s3 --save
```

Comment:
Assuming we're in the main Ghost install directory, this would modify Ghost's `package.json` to add our dependency. Not sure that it's a great idea to modify core files.